src/bridging/BridgingSdk/getCrossChainOrder.ts

```diff
@@ -0,0 +1,77 @@
+import { BridgeProvider, BridgeQuoteResult, BridgeStatus, CrossChainOrder } from '../types'
+import { SupportedChainId } from '../../chains'
+import { OrderBookApi } from 'src/order-book'
+import { getPostHooks } from '../utils'
+import { HOOK_DAPP_BRIDGE_PROVIDER_PREFIX } from '../providers/across/const/misc'
+
+/**
+ * Fetch a cross-chain order and its status.
+ */
+export async function getCrossChainOrder(params: {
+  orderId: string
+  chainId: SupportedChainId
+  orderBookApi: OrderBookApi
+  providers: BridgeProvider<BridgeQuoteResult>[]
+}): Promise<CrossChainOrder> {
+  const { orderId, chainId, orderBookApi, providers } = params
+
+  const chainContext = { chainId }
+  const order = await orderBookApi.getOrder(orderId, chainContext)
```

Comment:
We'll need to consider the env here as well (barn/prod)

Reply:
The env is part of the orderbookApi already: `orderBookApi.context.env`

If you want to use a different env, we can instanciate a different orderbook. Anyways, I added an optional `env` for convenience. Might be handy for the explorer 9a9701d8ccfc7f340a85dc1f3ef8f41ed7a1e826